server/controllers/article.js

```diff
@@ -150,6 +150,22 @@ class Article {
     });
     return goodHttpResponse(response, 201, 'Article rated', rating);
   }
+
+  /**
+   *
+   * @param {object} request Request Object
+   * @param {object} response Response Object
+   * @returns {object} response
+   */
+  static async deleteArticle(request, response) {
+    const { article, userId } = request;
+    const { userid } = article.dataValues;
+    if (userid === userId) {
+      await articleRepo.deleteArticle(article.dataValues);
+      return goodHttpResponse(response, 202, 'Article deleted');
```

Comment:
Is 202 the right status code for this?

Reply:
For a delete, we could a general 200 status or a specific 202 or 204. Since I am returning a message it cant be 204. [So 202 is the most specific](https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete).

Reply:
Yes it is. See [here](https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete)

Reply:
Ideally, this should be a 204 but since we're returning a payload 202 works as well.